aya/src/obj/mod.rs

```diff
@@ -706,32 +738,34 @@ mod tests {
                         value_size: 3,
                         max_entries: 4,
                         map_flags: 5,
-                        id: 0,
-                        pinning: 0
+                        ..Default::default()
                     })
                 ),
                 "foo"
             ),
             Ok(Map {
                 section_index: 0,
                 name,
-                def: bpf_map_def {
-                    map_type: 1,
-                    key_size: 2,
-                    value_size: 3,
-                    max_entries: 4,
-                    map_flags: 5,
-                    id: 0,
-                    pinning: 0
-                },
+                def,
                 data
             }) if name == "foo" && data.is_empty()
         ))
     }
 
     #[test]
     fn test_parse_map_data() {
+        #![allow(unused_variables)] // def is used by the assertion
```

Comment:
I don't think this test should be changed? Other than the Default::default() bit that is

Reply:
I was getting errors due to `_map_type` and `value_size` being undefined.

Reply:
Fixed in 86b639b